include/deal.II/matrix_free/matrix_free.h

```diff
@@ -725,6 +725,11 @@ class MatrixFree : public Subscriptor
      */
     values,
 
+    /**
+     * TODO
+     */
+    values_cell,
```

Comment:
Maybe:
```suggestion
    values_all_faces
```

Reply:
Excellent choice, yes, please use `values_all_faces` and `gradients_all_faces`.